ArduCopter/mode_acro.cpp

```diff
@@ -96,46 +96,34 @@ float ModeAcro::throttle_hover() const
 
 // get_pilot_desired_angle_rates - transform pilot's roll pitch and yaw input into a desired lean angle rates
 // returns desired angle rates in centi-degrees-per-second
-void ModeAcro::get_pilot_desired_angle_rates(int16_t roll_in, int16_t pitch_in, int16_t yaw_in, float &roll_out, float &pitch_out, float &yaw_out)
+void ModeAcro::get_pilot_desired_angle_rates(float roll_in, float pitch_in, float yaw_in, float &roll_out, float &pitch_out, float &yaw_out)
 {
+    // todo: provide normalised commands from RC library so this can be removed.
+    roll_in /= ROLL_PITCH_YAW_INPUT_MAX;
```

Comment:
Can these be inverted so that its a multiply?

Reply:
Yes but it should also be done back in the RC library and removed from here completly. So again I will stick with clarity.